frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/BreakoutColumnList/BreakoutColumnListItem/BreakoutColumnListItem.styled.tsx

```diff
@@ -108,12 +109,13 @@ const unselectedStyle = css`
       background-color: var(--mb-color-bg-medium);
     }
 
-    ${BucketPickerPopover.TriggerButton} {
+    .${BucketPickerPopoverStyle.triggerButton} {
+      visibility: visible;
```

Comment:
This style is required for when we're hovering over the add column button, the binning label should also be visible.
<img width="351" alt="Screenshot 2025-02-24 at 4 34 28 PM" src="https://github.com/user-attachments/assets/f93d6ce2-f82b-49cc-86a9-26b1be27ea4d" />